addons/forum-id/userscript.js

```diff
@@ -1,3 +1,4 @@
+/* global $, paste */
```

Comment:
I did `window.paste` because this addon uses `window.paste` everywhere except that one line